pyproject.toml

```
@@ -0,0 +1,130 @@
+#
+# Richie package
+#
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "richie"
+version = "2.24.1"
+description = "A CMS to build learning portals for open education"
+readme = "README.md"
+authors = [{ "name" = "Open FUN (France Université Numérique)", "email" = "fun.dev@fun-mooc.fr" }]
+license = { file = "LICENSE" }
+keywords = ["Django", "Django-CMS", "Open edX"]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Framework :: Django :: 1.11",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6"
+]
```

We could stick to [minimal requirements of Django CMS](https://github.com/django-cms/django-cms/blob/239dbf5d995bf3275db1dffe9eb55649e2a4a84a/setup.cfg#L27-L30) what do you think ?

```suggestion
classifiers = [
    "Development Status :: 5 - Production/Stable",
    "Framework :: Django",
    "Framework :: Django :: 2.2",
    "Intended Audience :: Developers",
    "License :: OSI Approved :: MIT License",
    "Natural Language :: English",
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3.7"
]
```